decoration.sk

```diff
@@ -664,8 +673,6 @@ nether update part 2:
 
 before caves and cliffs update part 1:
 	minecraft version = 1.16.5 or older
-	
-	(grass|dirt) path [block]¦s = minecraft:grass_path
```

Comment:
this needs restored (and likely added to `any dirt`)

should be the last thing 🙏

Reply:
It already exists unmodified on 741

Reply:
That's for 1.17 and above, we need to include it for 1.16 and below

Reply:
I didn't think I removed that

Reply:
replacement is still in the wrong spot :(